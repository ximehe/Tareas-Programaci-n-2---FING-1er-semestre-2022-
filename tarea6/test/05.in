# Prueba de operaciones de mapping
estaLlenoMap
esClaveEnMap 5
asociarEnMap 5 1.5
esClaveEnMap 5
estaLlenoMap
desasociarEnMap 5
esClaveEnMap 5
estaLlenoMap
asociarEnMap 5 2.5
esClaveEnMap 5
valorEnMap 5
asociarEnMap 10 1.2
asociarEnMap 20 1.3
asociarEnMap 90 1.4
asociarEnMap 80 1.5
asociarEnMap 70 1.6
asociarEnMap 60 1.7
asociarEnMap 40 1.8
asociarEnMap 30 1.9
valorEnMap 5
valorEnMap 30
estaLlenoMap
asociarEnMap 100 1.0
estaLlenoMap
reiniciarEstructuras
# Prueba de operaciones de mapping
esClaveEnMap 15
asociarEnMap 12 1.2
asociarEnMap 13 1.3
asociarEnMap 14 1.4
asociarEnMap 15 1.5
esClaveEnMap 15
estaLlenoMap
desasociarEnMap 15
esClaveEnMap 15
asociarEnMap 15 1.5
desasociarEnMap 15
esClaveEnMap 15
asociarEnMap 16 1.6
valorEnMap 12
valorEnMap 13
valorEnMap 14
valorEnMap 16
desasociarEnMap 12
desasociarEnMap 13
desasociarEnMap 14
desasociarEnMap 16
estaLlenoMap
reiniciarEstructuras
# Prueba esAvl
esAvl
insertarEnAbb (23,23.0)
esAvl
insertarEnAbb (15,15.0)
esAvl
insertarEnAbb (30,30.0)
insertarEnAbb (40,40.0)
esAvl
insertarEnAbb (10,10.0)
esAvl
insertarEnAbb (35,35.0)
esAvl
insertarEnAbb (45,45.0)
esAvl
insertarEnAbb (28,28.0)
esAvl
insertarEnAbb (50,50.0)
esAvl
insertarEnAbb (24,24.0)
esAvl
insertarEnAbb (5,5.0)
esAvl
insertarEnAbb (18,18.0)
esAvl
insertarEnAbb (4,4.0)
esAvl
insertarEnAbb (8,8.0)
esAvl
insertarEnAbb (12,12.0)
esAvl
insertarEnAbb (20,20.0)
esAvl
insertarEnAbb (11,11.0)
insertarEnAbb (13,13.0)
esAvl
insertarEnAbb (14,14.0)
esAvl
insertarEnAbb (21,21.0)
esAvl
insertarEnAbb (16,16.0)
esAvl
insertarEnAbb (9,9.0)
insertarEnAbb (6,6.0)
esAvl
insertarEnAbb (7,7.0)
esAvl
insertarEnAbb (2,2.0)
esAvl
insertarEnAbb (22,22.0)
esAvl
insertarEnAbb (19,19.0)
esAvl
insertarEnAbb (17,17.0)
esAvl
insertarEnAbb (55,55.0)
esAvl
insertarEnAbb (42,42.0)
esAvl
insertarEnAbb (37,37.0)
esAvl
insertarEnAbb (26,26.0)
esAvl
insertarEnAbb (29,29.0)
esAvl
reiniciarEstructuras
# Prueba avlMin
avlMin 1
avlMin 2
avlMin 3
avlMin 4
avlMin 5
avlMin 10
reiniciarEstructuras
# Prueba de filtradaOrdenada
filtradaOrdenada -1
filtradaOrdenada 5 5
insertarAlFinal 6 6.0
filtradaOrdenada -1
filtradaOrdenada 6 0
filtradaOrdenada 5 6 7 0
filtradaOrdenada 4 5 6 0
insertarAlFinal 4 4.0
insertarAlFinal 6 5.0
filtradaOrdenada 6 0
filtradaOrdenada 5 6 7 0
filtradaOrdenada 4 5 6 0
insertarAlFinal 5 5.0
insertarAlInicio 123456789 1.0
filtradaOrdenada 6 0
filtradaOrdenada 5 6 7 0
filtradaOrdenada 4 6 123456789 0
insertarAlFinal 33 33.0
insertarAlFinal 6 4.0
insertarAlFinal 700000 7.0
insertarAlFinal 4 3.0
insertarAlFinal 4 5.0
filtradaOrdenada 1 4 5 6 31 33 35 700000 123456789 0
reiniciarEstructuras
# Prueba de rotar
rotar 20 LR
insertarEnAbb (50,0)
rotar 20 LR
rotar 50 LR
rotar 50 LL
insertarEnAbb (30,0)
insertarEnAbb (40,0)
insertarEnAbb (45,0)
rotar 50 LL
imprimirAbb
rotar 30 RR
imprimirAbb
rotar 50 LR
imprimirAbb
rotar 40 RL
imprimirAbb
rotar 30 LL
imprimirAbb
rotar 40 LR
imprimirAbb
insertarEnAbb (35,0)
rotar 40 LR
imprimirAbb
insertarEnAbb (70,0)
insertarEnAbb (60,0)
insertarEnAbb (65,0)
insertarEnAbb (90,0)
insertarEnAbb (80,0)
insertarEnAbb (55,0)
imprimirAbb
rotar 70 RL
imprimirAbb
rotar 80 LL
imprimirAbb
rotar 70 RR
imprimirAbb
rotar 50 RL
imprimirAbb
rotar 70 LR
imprimirAbb
Fin