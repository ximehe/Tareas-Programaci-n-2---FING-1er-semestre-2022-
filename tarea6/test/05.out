1>#  Prueba de operaciones de mapping.
2>No lleno.
3>No existe asociacion.
4>Establecida la asociación.
5>Existe asociacion.
6>No lleno.
7>Eliminada la asociación.
8>No existe asociacion.
9>No lleno.
10>Establecida la asociación.
11>Existe asociacion.
12>2.50
13>Establecida la asociación.
14>Establecida la asociación.
15>Establecida la asociación.
16>Establecida la asociación.
17>Establecida la asociación.
18>Establecida la asociación.
19>Establecida la asociación.
20>Establecida la asociación.
21>2.50
22>1.90
23>No lleno.
24>Establecida la asociación.
25>LLeno.
26>Estructuras reiniciadas.
27>#  Prueba de operaciones de mapping.
28>No existe asociacion.
29>Establecida la asociación.
30>Establecida la asociación.
31>Establecida la asociación.
32>Establecida la asociación.
33>Existe asociacion.
34>No lleno.
35>Eliminada la asociación.
36>No existe asociacion.
37>Establecida la asociación.
38>Eliminada la asociación.
39>No existe asociacion.
40>Establecida la asociación.
41>1.20
42>1.30
43>1.40
44>1.60
45>Eliminada la asociación.
46>Eliminada la asociación.
47>Eliminada la asociación.
48>Eliminada la asociación.
49>No lleno.
50>Estructuras reiniciadas.
51>#  Prueba esAvl.
52>Es AVL.
53>Se insertó (23,23.00) en el abb.
54>Es AVL.
55>Se insertó (15,15.00) en el abb.
56>Es AVL.
57>Se insertó (30,30.00) en el abb.
58>Se insertó (40,40.00) en el abb.
59>Es AVL.
60>Se insertó (10,10.00) en el abb.
61>Es AVL.
62>Se insertó (35,35.00) en el abb.
63>NO es AVL.
64>Se insertó (45,45.00) en el abb.
65>NO es AVL.
66>Se insertó (28,28.00) en el abb.
67>Es AVL.
68>Se insertó (50,50.00) en el abb.
69>NO es AVL.
70>Se insertó (24,24.00) en el abb.
71>NO es AVL.
72>Se insertó (5,5.00) en el abb.
73>NO es AVL.
74>Se insertó (18,18.00) en el abb.
75>Es AVL.
76>Se insertó (4,4.00) en el abb.
77>NO es AVL.
78>Se insertó (8,8.00) en el abb.
79>NO es AVL.
80>Se insertó (12,12.00) en el abb.
81>NO es AVL.
82>Se insertó (20,20.00) en el abb.
83>Es AVL.
84>Se insertó (11,11.00) en el abb.
85>Se insertó (13,13.00) en el abb.
86>Es AVL.
87>Se insertó (14,14.00) en el abb.
88>NO es AVL.
89>Se insertó (21,21.00) en el abb.
90>NO es AVL.
91>Se insertó (16,16.00) en el abb.
92>Es AVL.
93>Se insertó (9,9.00) en el abb.
94>Se insertó (6,6.00) en el abb.
95>Es AVL.
96>Se insertó (7,7.00) en el abb.
97>NO es AVL.
98>Se insertó (2,2.00) en el abb.
99>NO es AVL.
100>Se insertó (22,22.00) en el abb.
101>NO es AVL.
102>Se insertó (19,19.00) en el abb.
103>NO es AVL.
104>Se insertó (17,17.00) en el abb.
105>NO es AVL.
106>Se insertó (55,55.00) en el abb.
107>NO es AVL.
108>Se insertó (42,42.00) en el abb.
109>NO es AVL.
110>Se insertó (37,37.00) en el abb.
111>NO es AVL.
112>Se insertó (26,26.00) en el abb.
113>NO es AVL.
114>Se insertó (29,29.00) en el abb.
115>Es AVL.
116>Estructuras reiniciadas.
117>#  Prueba avlMin.
118>(1,0.00)
119>(2,0.00)
-(1,0.00)
120>-(4,0.00)
(3,0.00)
-(2,0.00)
--(1,0.00)
121>-(7,0.00)
--(6,0.00)
(5,0.00)
--(4,0.00)
-(3,0.00)
--(2,0.00)
---(1,0.00)
122>--(12,0.00)
-(11,0.00)
--(10,0.00)
---(9,0.00)
(8,0.00)
--(7,0.00)
---(6,0.00)
-(5,0.00)
---(4,0.00)
--(3,0.00)
---(2,0.00)
----(1,0.00)
123>----(143,0.00)
-----(142,0.00)
---(141,0.00)
-----(140,0.00)
----(139,0.00)
-----(138,0.00)
------(137,0.00)
--(136,0.00)
-----(135,0.00)
----(134,0.00)
-----(133,0.00)
------(132,0.00)
---(131,0.00)
-----(130,0.00)
------(129,0.00)
----(128,0.00)
------(127,0.00)
-----(126,0.00)
------(125,0.00)
-------(124,0.00)
-(123,0.00)
-----(122,0.00)
----(121,0.00)
-----(120,0.00)
------(119,0.00)
---(118,0.00)
-----(117,0.00)
------(116,0.00)
----(115,0.00)
------(114,0.00)
-----(113,0.00)
------(112,0.00)
-------(111,0.00)
--(110,0.00)
-----(109,0.00)
------(108,0.00)
----(107,0.00)
------(106,0.00)
-----(105,0.00)
------(104,0.00)
-------(103,0.00)
---(102,0.00)
------(101,0.00)
-----(100,0.00)
------(99,0.00)
-------(98,0.00)
----(97,0.00)
------(96,0.00)
-------(95,0.00)
-----(94,0.00)
-------(93,0.00)
------(92,0.00)
-------(91,0.00)
--------(90,0.00)
(89,0.00)
-----(88,0.00)
----(87,0.00)
-----(86,0.00)
------(85,0.00)
---(84,0.00)
-----(83,0.00)
------(82,0.00)
----(81,0.00)
------(80,0.00)
-----(79,0.00)
------(78,0.00)
-------(77,0.00)
--(76,0.00)
-----(75,0.00)
------(74,0.00)
----(73,0.00)
------(72,0.00)
-----(71,0.00)
------(70,0.00)
-------(69,0.00)
---(68,0.00)
------(67,0.00)
-----(66,0.00)
------(65,0.00)
-------(64,0.00)
----(63,0.00)
------(62,0.00)
-------(61,0.00)
-----(60,0.00)
-------(59,0.00)
------(58,0.00)
-------(57,0.00)
--------(56,0.00)
-(55,0.00)
-----(54,0.00)
------(53,0.00)
----(52,0.00)
------(51,0.00)
-----(50,0.00)
------(49,0.00)
-------(48,0.00)
---(47,0.00)
------(46,0.00)
-----(45,0.00)
------(44,0.00)
-------(43,0.00)
----(42,0.00)
------(41,0.00)
-------(40,0.00)
-----(39,0.00)
-------(38,0.00)
------(37,0.00)
-------(36,0.00)
--------(35,0.00)
--(34,0.00)
------(33,0.00)
-----(32,0.00)
------(31,0.00)
-------(30,0.00)
----(29,0.00)
------(28,0.00)
-------(27,0.00)
-----(26,0.00)
-------(25,0.00)
------(24,0.00)
-------(23,0.00)
--------(22,0.00)
---(21,0.00)
------(20,0.00)
-------(19,0.00)
-----(18,0.00)
-------(17,0.00)
------(16,0.00)
-------(15,0.00)
--------(14,0.00)
----(13,0.00)
-------(12,0.00)
------(11,0.00)
-------(10,0.00)
--------(9,0.00)
-----(8,0.00)
-------(7,0.00)
--------(6,0.00)
------(5,0.00)
--------(4,0.00)
-------(3,0.00)
--------(2,0.00)
---------(1,0.00)
124>Estructuras reiniciadas.
125>#  Prueba de filtradaOrdenada.
126>
127>
128>Se insertó (6,6.00) al final de la cadena.
129>
130>(6,6.00)
131>(6,6.00)
132>(6,6.00)
133>Se insertó (4,4.00) al final de la cadena.
134>Se insertó (6,5.00) al final de la cadena.
135>(6,11.00)
136>(6,11.00)
137>(4,4.00)(6,11.00)
138>Se insertó (5,5.00) al final de la cadena.
139>Se insertó (123456789,1.00) al inicio de la cadena.
140>(6,11.00)
141>(5,5.00)(6,11.00)
142>(4,4.00)(6,11.00)(123456789,1.00)
143>Se insertó (33,33.00) al final de la cadena.
144>Se insertó (6,4.00) al final de la cadena.
145>Se insertó (700000,7.00) al final de la cadena.
146>Se insertó (4,3.00) al final de la cadena.
147>Se insertó (4,5.00) al final de la cadena.
148>(4,12.00)(5,5.00)(6,15.00)(33,33.00)(700000,7.00)(123456789,1.00)
149>Estructuras reiniciadas.
150>#  Prueba de rotar.
151>
152>Se insertó (50,0.00) en el abb.
153>
154>
155>
156>Se insertó (30,0.00) en el abb.
157>Se insertó (40,0.00) en el abb.
158>Se insertó (45,0.00) en el abb.
159>
160>
-(50,0.00)
---(45,0.00)
--(40,0.00)
(30,0.00)
161>
162>
(50,0.00)
---(45,0.00)
--(40,0.00)
-(30,0.00)
163>
164>
-(50,0.00)
--(45,0.00)
(40,0.00)
-(30,0.00)
165>
166>
-(50,0.00)
(45,0.00)
-(40,0.00)
--(30,0.00)
167>
168>
-(50,0.00)
(45,0.00)
-(40,0.00)
--(30,0.00)
169>
170>
-(50,0.00)
(45,0.00)
-(40,0.00)
--(30,0.00)
171>Se insertó (35,0.00) en el abb.
172>
173>
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
174>Se insertó (70,0.00) en el abb.
175>Se insertó (60,0.00) en el abb.
176>Se insertó (65,0.00) en el abb.
177>Se insertó (90,0.00) en el abb.
178>Se insertó (80,0.00) en el abb.
179>Se insertó (55,0.00) en el abb.
180>
---(90,0.00)
----(80,0.00)
--(70,0.00)
----(65,0.00)
---(60,0.00)
----(55,0.00)
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
181>
182>
---(90,0.00)
--(80,0.00)
---(70,0.00)
-----(65,0.00)
----(60,0.00)
-----(55,0.00)
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
183>
184>
----(90,0.00)
---(80,0.00)
--(70,0.00)
----(65,0.00)
---(60,0.00)
----(55,0.00)
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
185>
186>
---(90,0.00)
--(80,0.00)
---(70,0.00)
-----(65,0.00)
----(60,0.00)
-----(55,0.00)
-(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
187>
188>
---(90,0.00)
--(80,0.00)
-(70,0.00)
----(65,0.00)
---(60,0.00)
----(55,0.00)
--(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
189>
190>
----(90,0.00)
---(80,0.00)
--(70,0.00)
---(65,0.00)
-(60,0.00)
---(55,0.00)
--(50,0.00)
(45,0.00)
--(40,0.00)
-(35,0.00)
--(30,0.00)
191>Fin.
